mz_os_win32.c

```diff
@@ -569,8 +580,19 @@ int32_t mz_os_read_symlink(const char *path, char *target_path, int32_t max_targ
     if (path_wide == NULL)
         return MZ_PARAM_ERROR;
 
+#ifdef MZ_WINRT_API
+    CREATEFILE2_EXTENDED_PARAMETERS extendedParams = { 0 };
```

Comment:
Change variable name to `extended_params`.

Reply:
I assume I need to use memset here, too?

Reply:
Yes, thank you.

Reply:
Done